logrusr/logger.go

```diff
@@ -141,3 +104,16 @@ func New(name string, logger logrus.Logger) logr.Logger {
 		},
 	}
 }
+
+func createMap(kvs map[string]interface{}, extra []interface{}) map[string]interface{} {
+	ret := make(map[string]interface{})
+
+	for k, v := range kvs {
+		ret[k] = v
+	}
+
+	for i := 0; i < len(extra); i += 2 {
+		ret[extra[i].(string)] = extra[i+1]
+	}
+	return ret
+}
```

Comment:
I think this function could be simplified, and a range operation could be avoided.

```
	for i := 0; i < len(extra); i += 2 {
		kv[extra[i].(string)] = extra[i+1]
	}
	
	return kv
```